.github/workflows/lint.yml

```diff
@@ -12,7 +12,5 @@ jobs:
           node-version: '12.x'
       - name: Install dependencies
         run: npm ci
-      - name: Type Check
-        run: npm run typecheck
```

Comment:
I shouldn't have removed it indeed. I am thinking it might be better to run `npm run build` instead of `tsc --noEmit` to catch build errors that include type-checking errors. How do you think?

Reply:
sg